sidebars.js

```diff
@@ -6,15 +6,66 @@ const sidebars = {
     'docs/quickstart',
     {
       type: 'category',
-      label: 'Releases',
-      link: {type: 'doc', id: 'docs/releases'},
-      items: [{type: 'autogenerated', dirName: 'docs/releases'}],
+      label: 'Pomerium Core',
+      link: {type: 'doc', id: 'docs/pomerium-core'},
+      items: [
+        {
+          type: 'category',
+          label: 'Install',
+          items: [
+            {type: 'autogenerated', dirName: 'docs/pomerium-core/install'},
+          ],
+        },
+        'docs/pomerium-core/upgrading',
+        'docs/pomerium-core/changelog',
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Enterprise',
+      link: {type: 'doc', id: 'docs/enterprise'},
+      items: [
+        'docs/enterprise/quickstart',
+        {
+          type: 'category',
+          label: 'Install',
+          link: {type: 'doc', id: 'docs/enterprise/install/index'},
+          items: [
+            'docs/enterprise/install/docker',
+            'docs/enterprise/install/os-packages',
+            'docs/enterprise/install/helm',
+          ],
+        },
+        'docs/enterprise/configure',
+        'docs/enterprise/upgrading',
+        'docs/enterprise/changelog',
+      ],
+      // items: [{type: 'autogenerated', dirName: 'docs/enterprise'}],
+    },
+    {
+      type: 'category',
+      label: 'Kubernetes',
```

Comment:
Feels like this should live under "Deploy" thoughts?

Reply:
I moved "Kubernetes" because it has lots of pages, which increases the click depth to get to the right page. I was trying to avoid too many nested dropdowns, but it also adds an additional category to the sidebar. Can totally move this one under "Deploy", but also curious what @wasaga thinks?